vpr/src/route/rr_graph.cpp

```diff
@@ -861,41 +860,43 @@ static void load_rr_switch_inf(const int num_arch_switches, const float R_minW_n
             // setup device_ctx.switch_fanin_remap, for future swich usage analysis
             device_ctx.switch_fanin_remap[i_arch_switch][fanin] = i_rr_switch;
 
-            load_rr_switch_from_arch_switch(i_arch_switch, i_rr_switch, fanin, R_minW_nmos, R_minW_pmos);
+            load_rr_switch_from_arch_switch(i_arch_switch, fanin, R_minW_nmos, R_minW_pmos);
```

Comment:
Sorry. I just read my codes again. I think I gave a wrong suggestion before.

The function ``create_rr_switch_from_arch_switch()`` was used in a different way.
See the attached codes
https://github.com/lnis-uofu/OpenFPGA/blob/271c46d6acf77d9e14b0a70bc7a77346e0a716c3/vpr/src/route/rr_graph.cpp#L705-L716

Now, we should
- Add the codes https://github.com/lnis-uofu/OpenFPGA/blob/271c46d6acf77d9e14b0a70bc7a77346e0a716c3/vpr/src/route/rr_graph.cpp#L705-L716 to a similar place in  ``rr_graph.cpp`` 
- Revert back to ``load_rr_switch_from_arch_switch()`` (**keep the ``create_rr_switch_from_arch_switch()``**)